test/Elastica/Query/BoostingTest.php

```diff
@@ -16,15 +16,15 @@ class BoostingTest extends BaseTest
         ['name' => 'Vital Match', 'price' => 2.1],
         ['name' => 'Mercury Vital', 'price' => 7.5],
         ['name' => 'Fist Mercury', 'price' => 3.8],
-        ['name' => 'Lama Vital 2nd', 'price' => 3.2],
+        ['name' => 'Lama Vital 2nd', 'price' => 1.2],
```

Comment:
I was testing also the BoostingTest.php, and I forget to remove it during the creation of the PR.